x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/GrammarInDevelopmentParsingTests.java

```diff
@@ -31,7 +31,11 @@ public void testDevelopmentMatch() throws Exception {
     }
 
     public void testDevelopmentRerank() {
-        parse("row a = 1 | rerank \"foo\" ON title WITH reranker", "rerank");
+        parse("row a = 1 | rerank \"foo\" on title with reranker", "rerank");
+    }
+
+    public void testDevelopmentCompletion() {
+        parse("row a = 1 | completion concat(\"test\", \"a\" with inferenceId as fieldName", "completion");
```

Comment:
Are we not missing a parenthesis in concat() ? 🤔 

```suggestion
        parse("row a = 1 | completion concat(\"test\", \"a\") with inferenceId as fieldName", "completion");
```